package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "4.3.2",
+  "version": "4.3.3",
```

Comment:
It's good practise to update this in every PR, to more easily track features or breakages 😉